pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -411,7 +411,7 @@ func (m *kubeGenericRuntimeManager) makeMounts(opts *kubecontainer.RunContainerO
 			// open(2) to create the file, so the final mode used is "mode &
 			// ~umask". But we want to make sure the specified mode is used
 			// in the file no matter what the umask is.
-			if err := m.osInterface.Chmod(containerLogPath, 0666); err != nil {
+			if err := m.osInterface.Chmod(containerLogPath, 0660); err != nil {
```

Comment:
The PR mentions the terminationMessagePath that is deined a few lines below: https://github.com/kubernetes/kubernetes/pull/108076/files#diff-d86ffc24d8e4c9deafd3e39e8f4ee3546ee46a31dcea8ea6526443b3f4bffa28R420

How is it that changing the `containerLogPath` affects the `terminationMessagePath`?

Reply:
ok, so my goal is to get rid of the world writeable file regardless of how is it called. The error message on line 406 suggests that `termination-log file` is the same as `containerLogPath` - is it not?

The `terminationMessagePath` as I understand it is an absolute path that is used when mounting the volume and
it has not much to do with what is known as the concept of termination message despite the name. In fact, even the error message on 415 refers to `containerLogPath` rather than `terminationMessagePath`. 

At this point I am bit confused about this code - if that's not the line responsible for creating the empty world writable files like this:

```
# ls -la /var/lib/kubelet/pods/fa852268-2261-4ba4-9aaf-a4eaa98d1dfe/containers/kube-proxy/a62b249a
-rw-rw-rw- 1 root root 0 Mar 20 12:30 /var/lib/kubelet/pods/fa852268-2261-4ba4-9aaf-a4eaa98d1dfe/containers/kube-proxy/a62b249a
```

then what is?

Reply:
It might very well be the same, I'm unsure. But some explanation on the commit msg would be helpful so it is easier to review, you probably figured that out when writing the PR :)